src/hardware/hantek-6xxx/api.c

```diff
@@ -60,7 +60,7 @@ static const struct hantek_6xxx_profile dev_profiles[] = {
 		"Hantek", "6022BE", "hantek-6022be.fw",
 	},
 	{
-		0x8102, 0x8102, 0x04b5, 0x6022,
+		0x8102, 0x8102, 0x8102, 0xd120, //0x04b5, 0x6022, 
```

Comment:
hmm, it's not a total random id. I left the VID as it was 0x8102, but I assume that this is already a non registered VID to begin with...  So while I agree that an offical VID:PID from sigrok would be prefered, I don't think that it is a problem to use something else until we get one. I just don't like if me sainsmart shows up as a hantek...